app/javascript/components/pages/dashboard/container.spec.js

```diff
@@ -21,46 +20,57 @@ describe("<Dashboard />", () => {
     mountedComponent(<Dashboard />, {}, initialState);
     expect(screen.getAllByTestId("page-heading")).toHaveLength(1);
   });
-  it("should render a <PageHeading /> component", () => {
+
+  it("should render a navigation title", () => {
     mountedComponent(<Dashboard />, {}, initialState);
-    expect(screen.getByText(/navigation.home/i)).toBeInTheDocument();
+    expect(screen.getByText("navigation.home")).toBeInTheDocument();
   });
+
   it("should render a <PageContent /> component", () => {
     mountedComponent(<Dashboard />, {}, initialState);
     expect(screen.getByTestId("page-heading")).toBeInTheDocument();
   });
-  it("should render a <Overview /> component", () => {
+
+  it("should render a dashboard overview component", () => {
     mountedComponent(<Dashboard />, {}, initialState);
     expect(screen.getByText(/navigation.home/i)).toBeInTheDocument();
   });
+
   it("should render a <SharedFromMyTeam /> component", () => {
     mountedComponent(<Dashboard />, {}, initialState);
-    expect(screen.getByTestId("page-heading")).toBeInTheDocument();
+    expect(screen.getByText("navigation.home")).toBeInTheDocument();
```

Comment:
This will not tell us that the component `<SharedFromMyTeam>` was rendered. Each component should render a different title, I think this one should be: `dashboard.dash_shared_from_my_team`, so I'd search for that title instead:

`expect(screen.getByText("dashboard.dash_shared_from_my_team")).toBeInDocument()`

Reply:
The same should happen for the rest of the components in this file.

Reply:
Done